packages/friends/yml/en/stephen-crisp.yml

```diff
@@ -3,24 +3,28 @@ name: Stephen Crisp
 slug: stephen-crisp
 gender: male
 description:
-  Stephen Crisp (1628-1692) was another whose desperate search for Truth (lasting twenty
-  long years) at last brought him to an experiential knowledge of the grace and truth that
-  are in Jesus Christ. Having found the Pearl of great price, he sold all to buy it, and
-  soon was made an eminent minister among the early Society of Friends, laboring for the
-  next thirty-five years in the power and wisdom of the Spirit throughout England,
-  Scotland, Holland, and Germany.
+  Stephen Crisp (1628-1692) is another example of one whose desperate search for Truth
```

Comment:
I wonder if we should leave off the `another` here. It feels like it might lack context if you were just reading it on the back of a book. Another of whom? It makes a lot of sense if you're reading a bunch of descriptions like this, or have lots of descriptions on a web page, but I think if we consider it in isolation it should be left off.